tests/api.c

```diff
@@ -44847,6 +44847,34 @@ static void test_wolfSSL_EC_KEY_set_group(void)
 #endif
 }
 
+static void test_wolfSSL_EC_KEY_get_conv_form(void)
+{
+#if defined(HAVE_ECC) && defined(OPENSSL_EXTRA)
+    BIO* bio;
+    EC_KEY* key = NULL;
+
+    printf(testingFmt, "test_wolfSSL_EC_KEY_set_conv_form");
+
+    AssertIntLT(EC_KEY_get_conv_form(key), 0);
```

Comment:
Add test where NULL is passed to EC_KEY_get_conv_form().

Reply:
`key` is NULL here.

Reply:
Make it explicit. (Was fooled by the AssertIntLT instead of expected AssertIntEq.)